$MirOS: ports/emulators/qemu/patches/patch-target-mips_op_mem_c,v 1.1 2005/11/15 12:25:10 tg Exp $
--- target-mips/op_mem.c.orig	Tue Nov 15 12:17:53 2005
+++ target-mips/op_mem.c	Tue Nov 15 12:17:57 2005
@@ -18,6 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
+void dump_sc (void);
+
 /* Standard loads and stores */
 void glue(op_lb, MEMSUFFIX) (void)
 {
